src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -7,7 +7,7 @@ export interface ApiKeyModel {
     value?: string;
     displayName?: string;
     description?: string;
-    createdBy?: {};
+    createdBy?: object;

```

Comment:
I don't think we should use object. It's the same thing as `{}`. It throws an error "Property 'X' does not exist on type 'object'"

![image](https://user-images.githubusercontent.com/260007/68400678-6b6f0d80-0146-11ea-902f-5100566b4dda.png)

Reply:
`any` then? 😃